composer.json

```diff
@@ -59,5 +60,6 @@
         "branch-alias": {
             "dev-master": "0.3-dev"
```

Comment:
i suggest we move to 1.0 now, as per discussion in #3 this has been around for a long enough time now.

rather than branch-alias, we could set up a semantic branch 1.x for it, then we don't even need the alias anymore.